compiler/src/dotty/tools/dotc/transform/Splicer.scala

```diff
@@ -89,7 +89,6 @@ object Splicer {
          """.stripMargin
         ctx.error(msg, pos)
         None
-      case ex: Throwable => throw ex
```

Comment:
This looks wrong to me, a try/catch can always catch a `Throwable`.

Reply:
Yes, we could keep it, and no warning will be reported. I removed it because the compiler will generate a default case anyway.

Reply:
OK.